src/Microsoft.Framework.Runtime/FrameworkReferenceResolver.cs

```diff
@@ -22,6 +22,10 @@ public class FrameworkReferenceResolver : IFrameworkReferenceResolver
                     new FrameworkName(VersionUtility.NetFrameworkIdentifier, new Version(4, 5, 1))
                 }
             },
+            { new FrameworkName(VersionUtility.DnxFrameworkIdentifier, new Version(4, 5, 1)), new List<FrameworkName> {
```

Comment:
We can do this in a not so hardcoded way since we have 1:1 mappings with desktop now. Instead of hardcoding 4.5.1.

https://github.com/aspnet/XRE/blob/anurse/tfm-rename/src/Microsoft.Framework.Runtime/FrameworkReferenceResolver.cs#L236